Classes/Controller/ModuleController.php

```diff
@@ -220,16 +233,20 @@ public function editAccountAction(Account $account): void
      * Update a given account
      *
      * @param Account $account The account to update
+     * @param array $roleIdentifiers Identifiers of roles to assign to account
      * @param array $password Expects an array in the format array('<password>', '<password confirmation>')
      * @Flow\Validate(argumentName="password", type="\Neos\Neos\Validation\Validator\PasswordValidator", options={ "allowEmpty"=1, "minimum"=1, "maximum"=255 })
      * @return void
      * @throws NeosDomainException
      * @throws IllegalObjectTypeException
      * @throws UnsupportedRequestTypeException
      */
-    public function updateAccountAction(Account $account, array $password = []): void
+    public function updateAccountAction(Account $account, array $roleIdentifiers = [], array $password = []): void
     {
         if ($this->checkAccount($account)) {
+            // make sure self::$roleIdentifier is always added
```

Comment:
same here (?)

Reply:
Now only "FE user roles" can be assigned/removed. Possibly existing other roles are kept unchanged.